src/background/hypothesis-chrome-extension.js

```diff
@@ -171,21 +176,31 @@ export default function HypothesisChromeExtension({
     updateAnnotationCountIfEnabled(tabId, url);
   }
 
-  // This function will be called multiple times as the tab reloads.
-  // https://developer.chrome.com/extensions/tabs#event-onUpdated
-  //
-  // 'changeInfo' contains details of what changed about the tab's status.
-  // Two important events are when the tab's `status` changes to `loading`
-  // when the user begins a new navigation and when the tab's status changes
-  // to `complete` after the user completes a navigation
-  function onTabUpdated(tabId, changeInfo, tab) {
-    if (changeInfo.status === TAB_STATUS_LOADING) {
-      resetTabState(tabId, tab.url);
-      const query = directLinkQuery(tab.url);
+  /**
+   * This function will be called multiple times as the tab reloads.
+   * https://developer.chrome.com/extensions/tabs#event-onUpdated
+   *
+   * 'changeInfo' contains details of what changed about the tab's status.
+   * Two important events are when the tab's `status` changes to `loading`
+   * when the user begins a new navigation and when the tab's status changes
+   * to `complete` after the user completes a navigation
+   *
+   * @param {number} tabId
+   * @param {chrome.tabs.TabChangeInfo} changeInfo
+   * @param {chrome.tabs.Tab} tab
+   */
+  function onTabUpdated(tabId, { status }, tab) {
+    const url = /** @type {string} - property included because manifest has the "tabs" permission*/ (tab.url);
```

Comment:
I think it would be slightly more readable if the "property included ..." comment was moved to a separate line above.